api/types/container/deprecated.go

```diff
@@ -0,0 +1,16 @@
+package container // import "github.com/docker/docker/api/types/container"
```

Comment:
Wandering if we really need these? The interfaces changed and these aren't used any more so I fail to see why we still have them.

Reply:
I usually have a quick peek if there's external consumers (e.g. https://grep.app/search?q=.ContainerCreateCreatedBody), and if there are, I add the "deprecated" alias, so that linters will give consumers a way to find what to use instead, but remove the alias after one release (having given them time to migrate the code).